armada/api/middleware.py

```diff
@@ -12,96 +12,87 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from armada.conf import default
 
-# Required Oslo configuration setup
+from armada.conf import default
 default.register_opts()
 
-from keystoneauth1 import session
-from keystoneauth1.identity import v3
+import falcon
+import re
+import uuid
 
 from oslo_config import cfg
 from oslo_log import log as logging
 
-import falcon
-
 LOG = logging.getLogger(__name__)
 CONF = cfg.CONF
-DOMAIN = "armada"
+DOMAIN = 'armada'
 
 logging.setup(CONF, DOMAIN)
 
 class AuthMiddleware(object):
 
     def process_request(self, req, resp):
+        ctx = req.context
+        LOG.info(ctx)
 
-        # Validate token and get user session
-        token = req.get_header('X-Auth-Token')
-        req.context['session'] = self._get_user_session(token)
+        for k, v in req.headers.items():
+            LOG.debug("Request with header %s: %s" % (k, v))
```

Comment:
Once you've confirmed Keystone is working, you may want to remove this statement. It is pretty noisy even for debug output.